extensions/container-image/container-image-jib/deployment/src/main/java/io/quarkus/container/image/jib/deployment/JibProcessor.java

```diff
@@ -203,7 +203,7 @@ private JibContainerBuilder createContainerBuilderFromJar(JibConfig jibConfig,
             }
 
             return javaContainerBuilder.toContainerBuilder()
-                    .setEnvironment(jibConfig.environmentVariables.orElse(Collections.emptyMap()))
+                    .setEnvironment(jibConfig.environmentVariables)
```

Comment:
`jibConfig.environmentVariables` will be null if it is not set.

Reply:
It will not 😎. I tried it and it's an empty map. But let me try it again to be 100% sure

Reply:
In general, Quarkus config objects never have null values.

Here is the screenshot from the debugger for a test I did:

![Screenshot from 2020-03-14 14-28-05](https://user-images.githubusercontent.com/4374975/76681905-34cb6700-6600-11ea-9473-c82536b7fa5b.png)

Reply:
Hm I could swear I saw it being null in https://github.com/quarkusio/quarkus/blob/master/extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java#L113

But maybe because of this: https://github.com/quarkusio/quarkus/blob/master/extensions/flyway/runtime/src/main/java/io/quarkus/flyway/runtime/FlywayDataSourceRuntimeConfig.java#L20

Thanks for proving me wrong, I was annoyed by that ;)

Reply:
Yeah, I think you found the cause of the `null` value 👍